s3.py

```diff
@@ -102,6 +102,12 @@ class S3AddressingStyle(Enum):
 class S3Uploader(MirrorUploader):
     NAME = ExternalIntegration.S3
 
+    # AWS S3 host
+    S3_HOST = 'amazonaws.com'
+
+    # MinIO host used in integration tests (we assume that it's always using port 9000)
+    MINIO_HOST = 'localhost:9000'
```

Comment:
I think the hostname could also be pulled from an environment variable.

Reply:
> We could set two attributes: `integration` and `minio` so that you can run the tests with `nosetests -a '!integration'` to run all the tests except those requiring external integration? Thoughts @leonardr or @tdilauro?

Yeah, this makes sense to me. It would be very important to me to be able to easily run the tests without any of the integrations and I think this approach would work nicely.

Reply:
I replaced the hard-coded with an environment variable